lib/pageflow/configuration/permissions.rb

```diff
@@ -0,0 +1,30 @@
+module Pageflow
+  class Configuration
+    # Permissions related options to be defined in the pageflow
+    # initializer of the main app.
+    class Permissions
+      def initialize(options = {})
+        @show_entry_features_to_non_admin =
+          options.fetch(:show_entry_features_to_non_admin, true)
+        @show_admin_boolean_to_non_admin =
+          options.fetch(:show_admin_boolean_to_non_admin, true)
+        @show_theming_dropdown_to_non_admin =
+          options.fetch(:show_theming_dropdown_to_non_admin, true)
+      end
+
+      # Allow non-admin users to see entry features. Defaults to true.
+      # @since edge
+      attr_accessor :show_entry_features_to_non_admin
```

Comment:
I do not really care about "may" vs "can" as long as we stay consistent.

Regarding the first bullet point, I disagree. If we ever adjust permissions such that other roles besides account managers may update features (which I have a hard time imagining at the moment), simply changing the meaning of this option would be a rather nasty breaking change IMO. We should strive to make this permission interface as explicit as possible.